### PR TITLE
refact(api): refact workflowrun status

### PR DESCRIPTION
**Pull request by supereagle**
**What this PR does / why we need it**:

Refactor workflowrun status to make it clearer.

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @cd1989  @zhujian7 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by zhujian7** [comment]
@qiuxiaolian PTAL

File: pkg/server/biz/statistic/statistic.go [hunk 1]
```diff
@@ -126,9 +126,9 @@ func formatTimeToDay(t time.Time) int64 {
 
 func statsStatus(s api.StatsPhase, recordStatus v1alpha1.StatusPhase) api.StatsPhase {
 	switch recordStatus {
-	case v1alpha1.StatusCompleted:
+	case v1alpha1.StatusSucceeded:
 		s.Completed++
```
Comment (cd1989) [inline pkg/server/biz/statistic/statistic.go]: Do we need also change this ?

Comment (supereagle) [reply pkg/server/biz/statistic/statistic.go]: Done

File: pkg/workflow/workflowrun/operator.go [hunk 2]
```diff
@@ -333,7 +333,7 @@ func (o *operator) Reconcile() error {
 			log.WithField("wfr", o.wfr.Name).WithField("stg", stage).Error("Create pod manifest for stage error: ", err)
 			o.recorder.Eventf(o.wfr, corev1.EventTypeWarning, "GeneratePodSpecError", "Generate pod for stage '%s' error: %v", stage, err)
 			o.UpdateStageStatus(stage, &v1alpha1.Status{
-				Phase:              v1alpha1.StatusError,
+				Phase:              v1alpha1.StatusFailed,
 				Reason:             "GeneratePodError",
```
Comment (cd1989) [inline pkg/workflow/workflowrun/operator.go]: `GeneratePodError ` -- > `GeneratePodFailed `

Comment (supereagle) [reply pkg/workflow/workflowrun/operator.go]: Will normalize these reasons later.

**Pull request merged**